### PR TITLE
Leaderboard: final touches

### PR DESCRIPTION
**Pull request by benjp009**
# 👋 Micronaut Reviewer 

[Leaderboard: final touches](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m3_final_touches.md)

Screenshot 2021-12-17 at 13 16 57

### Project requirements

- [x] Now you should improve the look and feel of the application, adding the styles of your choice.
- [x] Please keep the general layout of the wireframe, this is the only mandatory requirement.
- [x] You can use plain CSS or any CSS framework, it's up to you.

### Pair Programming 🎳

For this exercise, PP was not equired.

Happy review :)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -18,22 +19,21 @@ const sendData = async (url = '', data = {}) => {
...
+const scoresUl = document.getElementsByClassName('boardList')[0];
 
 const loadScores = async () => {
   scoresUl.innerHTML = '';
   const { result: scores } = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/3qxAA7wy5DPzg65q4nWs/scores/').then((response) => response.json());
   scores.forEach((score) => {
     const li = document.createElement('li');
-    li.innerHTML = `${score.user}: ${score.score}`;
+    li.innerHTML = `${score.user} : ${score.score}`;
     scoresUl.appendChild(li);
```
Comment (Dipeshtwis) [inline src/index.js]: - [x] I am not able to send and receive data. Please make sure that submit button is working well and hitting the API.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @benjp009, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: The project style is looking good.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

- [x] In the README file, Attach your project screenshot instead of a wireframe screenshot so that any user or recruiter can see your project through pictures as well.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (benjp009) [reply src/index.js]: Changes done :)
Thanks for your review

**Review by oloomoses — APPROVED** [review]
Hi @benjp009, 

Well done on making the suggested changes.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by benjp009** [comment]
@oloomoses  Thanks for your review :) 
Happy Weekend

**Pull request merged**